addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -388,7 +388,7 @@ private void handleCancelEvent(JSONObject data) {
                 runningRequests.remove(requestId);
             }
         } catch (JSONException e) {
-            logger.error(e.getMessage());
+            logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace